app/containers/Avatar.js

```diff
@@ -23,7 +24,17 @@ const Avatar = React.memo(({
 		type, text, size, userId, token, avatar, baseUrl
 	});
 
-	let image = (
+	let image = emoji ? (
+		<Emoji
+			theme={theme}
+			baseUrl={baseUrl}
+			getCustomEmoji={getCustomEmoji}
+			isMessageContainsOnlyEmoji
+			customEmojis
+			literal={emoji}
+			emojiName={emoji.replace(/:/g, '')}
```

Comment:
I think you can change `Markdown/Emoji` to do this logic inside there, receive only the `literal` prop and add `:` to start and end when you try to `getCustomEmoji`.
Remember to change this inside `Markdown/index`.
Thanks.

Reply:
So, I have passed only `literal` prop and then used `literal.replace(/:/g, '')` in `getCustomEmoji` to get `emoji` and made the changes in the `Markdown/index` to only pass `literal`